server/controllers/board.js

```diff
@@ -29,13 +30,32 @@ exports.updateBoard = asyncHandler(async (req, res, next) => {
   }
 });
 
+// @route PUT /board/create
+// @desc Creates new board with given title to given user
+// @access Private
+exports.createBoard = asyncHandler(async (req, res, next) => {
+  try {
+    const { id, title } = req.body;
+    const userModel = await User.findById(id).exec();
+    const board = await generateBoard(title);
+    userModel.boards.push(board);
+    await User.findByIdAndUpdate(id, userModel);
```

Comment:
While this works, you could also just save off the userModel. `await userModel.save()`